### PR TITLE
Prohibit frame components in times

### LINKED ISSUE #123: Consider restricting time expressions
Given that the time expression syntax for `#time-clock-with-frames` can be misinterpreted as a SMPTE time code, which are not supported, we should consider not permitting `#time-clock-with-frames` and only support `hh:mm:ss.ms` or `f`.

### PR DESCRIPTION
**Pull request by nigelmegitt**
Closes #123.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/dapt/pull/127.html" title="Last updated on Apr 10, 2023, 12:58 PM UTC (14589e1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/dapt/127/daf097f...14589e1.html" title="Last updated on Apr 10, 2023, 12:58 PM UTC (14589e1)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1554,32 +1554,11 @@ <h4><code>timeContainer</code></h4>
             the computed begin time of its parent element,
             or for the <code>&lt;body&gt;</code> element, to time zero.</p>
         </section> <!-- timecontainer-section -->
-        <section id="framerate-section">
-          <h4><code>ttp:frameRate</code></h4>
-          <p>If the document contains any time expression that uses the <code>f</code> metric,
-            or any time expression that contains a frames component,
-            the <code>ttp:frameRate</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element.</p>
-        </section> <!-- framerate-section -->
```
Comment (cconcolato) [inline index.html:1562]: I'm not sure we want to remove `f`. The only thing we want to remove is the syntax that looks like SMPTE time code.

File: index.html [hunk 2]
```diff
@@ -2177,13 +2156,13 @@ <h4>Disposition of Features and Extensions</h4>
...
-            <td>See [[[#framerate-section]]] and [[[#time-clock-with-frames-section]]].</td>
+            <td><span class="prohibited label">prohibited</span></td>
+            <td></td>
           </tr>
           <tr>
             <td><code>#time-offset-with-frames</code></td>
-            <td><span class="permitted label">permitted</span></td>
-            <td>See [[[#framerate-section]]].</td>
+            <td><span class="prohibited label">prohibited</span></td>
+            <td></td>
```
Comment (cconcolato) [inline index.html:2165]: This one should still be allowed

File: index.html (hunk 1 above)
Comment (nigelmegitt) [reply index.html:1562]: Seems like unnecessary complexity to support frame counts if no other frame based expressions are permitted. Why not use ticks if an arbitrary rate time counting mechanism is needed?

I've realised I am not sure if subframes have any bearing on frame count time expressions.

Comment (nigelmegitt) [reply index.html:1562]: I see from the issue that the proposal was to retain `f` units so I should have checked more carefully. Will fix when I can.

File: index.html [hunk 3]
```diff
@@ -1590,19 +1575,23 @@ <h4>Time expressions</h4>
             <ul>
```
Comment (cconcolato) [inline index.html:1575]: Maybe we should say "A clock-time as restricted by this specification"?

**Review by cconcolato — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (nigelmegitt) [reply index.html:1575]: Equivalent done in 14589e1.

**Pull request merged**